Storage/SQL/SQLiteHistoryRecommendations.swift

```diff
@@ -10,6 +10,15 @@ import Deferred
 fileprivate let log = Logger.syncLogger
 
 extension SQLiteHistory: HistoryRecommendations {
+    // These numbers will likely need additional tweaking in the future.
+    // They are currently based on a best-guess as to where performance
+    // starts to suffer in a large browser.db. A database with 200,001
+    // unique history items with 20 visits each produces a 340MB file.
```

Comment:
FWIW, history visits aren't uniformly distributed. It doesn't matter too much, though: 200,001 history items with zero visits each won't be too much smaller, because it's the strings that do it.